SimpleInventoryManagementSystem/Product.cs

```diff
@@ -1,19 +1,18 @@
 ﻿
 using System;
 
-namespace ISimpleInventoryManagementSystem
+namespace SimpleInventoryManagementSystem;
+
+public class Product
 {
-    public class Product
-    {
-        public string Name { get; set; }
-        public decimal Price { get; set; }
-        public int Quantity { get; set; }
+    public string _name { get; set; }
+    public float _price { get; set; }
+    public int _quantity { get; set; }
```

Comment:
We can get rid of the constructor if we make the properties `required`. Also, properties are Pascal cased.

```suggestion
    public string Name { get; set; }
    public float Price { get; set; }
    public int Quantity { get; set; }
```
Final result:
```csharp
public class Product
{
    public required string Name { get; set; }
    public required float Price { get; set; }
    public required int Quantity { get; set; }
}
```